APISpecification.yaml

```diff
@@ -53,6 +53,63 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenomeDetails'
+  /api/metadata/genome/{genome_id}/karyotype:
+    get:
+      summary: Returns a list of top-level regions included in the organism's karyotype
+      parameters:
+      - name: genome_id
+        in: query
+        required: true
+        schema:
+          type: string
+          example: 3704ceb1-948d-11ec-a39d-005056b38ce3
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Karyotype'
+        404:
+          description: Genome not found
+          content: {}
+  /api/metadata/validate_region:
+    get:
+      summary: Checks validity of user input containing a genomic region
+      parameters:
+      - name: genome_id
+        in: query
+        required: true
+        schema:
+          type: string
+          example: 3704ceb1-948d-11ec-a39d-005056b38ce3
+      - name: region
+        in: query
+        description: User input string for validation
+        required: true
+        schema:
+          type: string
+          example: 1:10000-1000000
+      responses:
+        200:
+          description: region validation information (region can be either valid or invalid)
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RegionValidationResponse'
+        400:
+          description: bad input; for example due to parsing error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: object
+                    example:
+                      parse: "Could not parse region 13 32,442,859-32,272,495"
```

Comment:
Not sure how to properly represent this error that I am seeing returned from the endpoint in ensembl-2020-genome-search api:

```
{
  "message": {
    "parse": "Could not parse region 13 32,442,859-32,272,495"
  }
}
```